lib/modules/datasource/metadata.spec.ts

```diff
@@ -256,4 +256,93 @@ describe('modules/datasource/metadata', () => {
       'https://example.com/foo/bar'
     );
   });
+
+  it('Should delete homepage if its not a link to a path in github repo', () => {
+    const dep = {
+      homepage: 'http://github.com/foo',
+      releases: [
+        { version: '1.0.1', releaseTimestamp: '2000-01-01T12:34:56' },
+        { version: '1.0.2', releaseTimestamp: '2000-01-02T12:34:56.000Z' },
+        { version: '1.0.3', releaseTimestamp: '2000-01-03T14:34:56.000+02:00' },
+      ],
+    };
+    addMetaData(dep, MavenDatasource.id, 'foobar');
+    expect(dep).toMatchObject({
+      releases: [
+        {
+          version: '1.0.1',
+          releaseTimestamp: '2000-01-01T12:34:56.000Z',
+        },
+        {
+          version: '1.0.2',
+          releaseTimestamp: '2000-01-02T12:34:56.000Z',
+        },
+        {
+          version: '1.0.3',
+          releaseTimestamp: '2000-01-03T12:34:56.000Z',
+        },
+      ],
+      sourceUrl: 'http://github.com/foo',
```

Comment:
Also not valid source url

Reply:
This test is essentially a different case of of this test : https://github.com/renovatebot/renovate/blob/46e856418be8242ef8de6afc517f168d89310c02/lib/modules/datasource/metadata.spec.ts#L188-L200

Where `homepage` value needs to be moved to `sourceUrl`

Reply:
```suggestion
      sourceUrl: 'http://github.com/foo/bar',
```

Should at least have a valid org and repo